packages/core/src/request-builder/response-transformers.ts

```diff
@@ -6,6 +6,12 @@ import { Constructable } from '../constructable';
 import { Entity } from '../entity';
 import { deserializeEntity } from '../entity-deserializer';
 
+export function transformReturnValueForPromiseVoid<ReturnT>(data: any, builderFn: (data: any) => ReturnT) {
+  return new Promise<void>(resolve => {
+    resolve();
+  });
+}
+
```

Comment:
Why promise? Isn't this what we rather want?
```suggestion
export function transformReturnValueForVoid(data: any, builderFn: (data: any) => undefined): void {
  return builderFn(data);
}

```

Reply:
yes, but the I needed to make the method async so that undefined is wrapped in a promise. I change it to this form.